roles/simple_bind/tasks/main.yaml

```
@@ -6,6 +6,14 @@
     name: bind9
     state: present
 
+- name: Find installed version
+  ansible.builtin.command:
+    argv: [/usr/sbin/named, -V]
```

Perhaps

```suggestion
    argv: [/usr/sbin/named, -v]
```

is enough? Or is it missing in older versions?

---

Oops, I didn't notice it existed :)